app/models/request_type.rb

```diff
@@ -60,8 +60,8 @@ class RequestTypePlatePurpose < ApplicationRecord # rubocop:todo Style/Documenta
 
   # Defines the acceptable plate purposes or the request type.  Essentially this is used to limit the
   # cherrypick plate types when going into pulldown to the correct list.
-  has_many :plate_purposes, class_name: 'RequestType::RequestTypePlatePurpose'
-  has_many :acceptable_plate_purposes, through: :plate_purposes, source: :plate_purpose
+  has_many :purposes, class_name: 'RequestType::RequestTypePlatePurpose'
+  has_many :acceptable_plate_purposes, through: :purposes, source: :purpose
```

Comment:
These changes are needed in order to add the Cardinal request types, because they come from a tube rather than a plate. Without these changes, the request type record loader errors on `request_type.acceptable_plate_purposes << Purpose.find_by!(name: name)`